tests/std/tests/P0645R10_text_formatting_parsing/test.cpp

```diff
@@ -200,13 +219,53 @@ constexpr bool test_parse_precision() {
     return true;
 }
 
+template <typename CharT>
+constexpr bool test_parse_format_specs() {
+    auto parse_format_specs_fn = _Parse_format_specs<CharT, testing_callbacks<CharT>>;
+    using view_typ             = basic_string_view<CharT>;
+
+    view_typ s0(TYPED_LITERAL(CharT, "6}"));
+    view_typ s1(TYPED_LITERAL(CharT, "*<6"));
+    view_typ s2(TYPED_LITERAL(CharT, "*>6}"));
+    view_typ s3(TYPED_LITERAL(CharT, "*^6}"));
+    view_typ s4(TYPED_LITERAL(CharT, "6d}"));
+    view_typ s5(TYPED_LITERAL(CharT, "*^+4.4a}"));
+    view_typ s6(TYPED_LITERAL(CharT, "*^+#04.4a}"));
+    test_parse_helper(parse_format_specs_fn, s0, false, s0.size() - 1, {.expected_width = 6});
+    test_parse_helper(parse_format_specs_fn, s1, false, s1.size(),
+        {.expected_alignment = _Align::_Left,
+            .expected_fill   = view_typ(TYPED_LITERAL(CharT, "*")),
+            .expected_width  = 6});
+    test_parse_helper(parse_format_specs_fn, s2, false, s2.size() - 1,
+        {.expected_alignment = _Align::_Right,
+            .expected_fill   = view_typ(TYPED_LITERAL(CharT, "*")),
+            .expected_width  = 6});
+    test_parse_helper(parse_format_specs_fn, s3, false, s3.size() - 1,
+        {.expected_alignment = _Align::_Center,
+            .expected_fill   = view_typ(TYPED_LITERAL(CharT, "*")),
+            .expected_width  = 6});
+    test_parse_helper(parse_format_specs_fn, s4, false, s4.size() - 1, {.expected_width = 6, .expected_type = 'd'});
+    test_parse_helper(parse_format_specs_fn, s5, false, s5.size() - 1,
+        {.expected_alignment    = _Align::_Center,
+            .expected_sign      = _Sign::_Plus,
+            .expected_fill      = view_typ(TYPED_LITERAL(CharT, "*")),
+            .expected_width     = 4,
+            .expected_precision = 4,
+            .expected_type      = 'a'});
+    test_parse_helper(parse_format_specs_fn, s6, false, s6.size() - 1,
+        {.expected_alignment    = _Align::_Center,
+            .expected_sign      = _Sign::_Plus,
+            .expected_fill      = view_typ(TYPED_LITERAL(CharT, "*")),
+            .expected_width     = 4,
+            .expected_precision = 4,
+            .expected_hash      = true,
+            .expected_zero      = true,
+            .expected_type      = 'a'});
+    return true;
+}
+
 int main() {
     test_parse_align<char>();
-    test_parse_align<wchar_t>();
-    static_assert(test_parse_align<char>());
-    static_assert(test_parse_align<wchar_t>());
-    test_parse_arg_id<char>();
-    test_parse_arg_id<wchar_t>();

```

Comment:
Why are we removing this coverage?

Reply:
possibly a bad rebase, will take a look

Reply:
yeah, bad rebase. I added some newlines to hopefully make git handle main() better, and I'll be more careful in the future.